Makefile

```diff
@@ -127,13 +128,35 @@ docker-push:
 	@echo "Pushing docker image: $(DOCKER_TAG)"
 	@-docker push "$(DOCKER_TAG)"
 
+.PHONY: docker-stop
+docker-stop:
+	@echo "Stopping test docker container: $(DOCKER_TEST)"
+	@-docker container stop "$(DOCKER_TEST)" 2>/dev/null || true
+	@-docker rm $(DOCKER_TEST) 2>/dev/null || true
+
+.PHONY: docker-test
+docker-test: docker-build docker-stop
+	@echo "Smoke test of docker container: $(DOCKER_TAG)"
```

Comment:
It was correctly "docker image" here.  So we are testing the image `$(DOCKER_TAG)` but when we stop, we stop the container `$(DOCKER_TEST)`.